doc/developer/server_side.rst.tmpl

```diff
@@ -53,18 +53,18 @@ Before running the tests, install and build all dependencies:
 
 .. prompt:: bash
 
-    MAJOR_VERSION=${MAJOR_VERSION} ci/preparetest
+    make preparetest
 
 Run the tests:
 
 .. prompt:: bash
 
     docker-compose exec tests pytest --verbose --color=yes \
-        /app/c2cgeoportal/commons/tests
+        /opt/c2cgeoportal/commons/tests
     docker-compose exec tests pytest --verbose --color=yes \
-        /app/c2cgeoportal/geoportal/tests
+        /opt/c2cgeoportal/geoportal/tests
     docker-compose exec tests pytest --verbose --color=yes \
-        /app/c2cgeoportal/admin/tests
+        /opt/c2cgeoportal/admin/tests
     docker-compose exec qgisserver-tests pytest --verbose --color=yes \
         /src/tests/functional
```

Comment:
Why not some test targets ? For example: 
```
.PHONY: tests
tests: test_commons  test_geoportal test_admin
```
or something similar.

Reply:
+1 in an next pull request :-)